### PR TITLE
Bug 1457114, updated downgrading docs (3.5)

### PR DESCRIPTION
**Pull request by ahardin-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1457114

Preview Build: http://file.rdu.redhat.com/~ahardin/06212017/downgrade-steps/install_config/downgrade.html

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@mtnbikenc PTAL at my first pass at updating the Downgrading docs.

**Comment by ahardin-rh** [comment]
@adellape PTAL

**Comment by ahardin-rh** [comment]
@mtnbikenc PTAL

File: install_config/downgrade.adoc [hunk 1]
```diff
@@ -130,26 +116,22 @@ remove the following packages:
     atomic-openshift-sdn-ovs \
     tuned-profiles-atomic-openshift-node
```
Comment (adellape) [inline install_config/downgrade.adoc]: Add:

atomic-openshift-excluder
atomic-openshift-docker-excluder

File: install_config/downgrade.adoc [hunk 2]
```diff
@@ -222,29 +201,28 @@ On each master, install the following packages:
     openvswitch \
     atomic-openshift-sdn-ovs \
     tuned-profiles-atomic-openshift-node
+    atomic-openshift-excluder
+    atomic-openshift-docker-excluder
```
Comment (adellape) [inline install_config/downgrade.adoc]: Each line needs a ` \` at the end except for the last one.

File: install_config/downgrade.adoc [hunk 3]
```diff
@@ -222,29 +201,28 @@ On each master, install the following packages:
...
 
-====
 ----
 # yum install atomic-openshift \
     atomic-openshift-node \
     openvswitch \
     atomic-openshift-sdn-ovs \
     tuned-profiles-atomic-openshift-node
+    atomic-openshift-excluder
+    atomic-openshift-docker-excluder
```
Comment (adellape) [inline install_config/downgrade.adoc]: Same re: `\`

File: install_config/downgrade.adoc [hunk 4]
```diff
@@ -262,40 +240,34 @@ and Restore].
...
 and *router* images, if deployed:
-
-====
++
 ----
 ifdef::openshift-enterprise[]
 # oc get -n default dc/docker-registry -o json | grep \"image\"
     "image": "openshift3/ose-docker-registry:v3.1.1.6",
 # oc get -n default dc/router -o json | grep \"image\"
     "image": "openshift3/ose-haproxy-router:v3.1.1.6",
```
Comment (adellape) [inline install_config/downgrade.adoc]: Probably should update these 3.1 image tags to show the latest versions of these images for 3.4 instead.

**Comment by ahardin-rh** [comment]
@adellape Thanks! Updated.

**Comment by ahardin-rh** [comment]
QE verified

**Comment by ahardin-rh** [comment]
[rev_history]
|xref:../install_config/downgrade.adoc#install-config-downgrade[Downgrading OpenShift]
|Updated the downgrading steps for clarity.
%

**Pull request merged**